### ISSUE TITLE
Are there problems installing on M-series Macs? 

### ISSUE THREAD

**Issue by jmbhughes** [punch-mission/regularizepsf#62]
It looks to somehow be related to hdf5. Maybe I just haven't set up my laptop completely?

I got this when I tried:

```
Collecting regularizepsf
  Using cached regularizepsf-0.2.1.tar.gz (80 kB)
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
  Preparing metadata (pyproject.toml): started
  Preparing metadata (pyproject.toml): finished with status 'done'
Requirement already satisfied: astropy in /Users/jhughes/Desktop/repos/punchbowl/venv/lib/python3.9/site-packages (from regularizepsf) (5.3.3)
Collecting sep
  Using cached sep-1.2.1-cp39-cp39-macosx_11_0_arm64.whl (230 kB)
Collecting cython
  Using cached Cython-3.0.2-py2.py3-none-any.whl (1.2 MB)
Requirement already satisfied: matplotlib in /Users/jhughes/Desktop/repos/punchbowl/venv/lib/python3.9/site-packages (from regularizepsf) (3.8.0)
Collecting lmfit
  Using cached lmfit-1.2.2-py3-none-any.whl (102 kB)
Requirement already satisfied: scipy in /Users/jhughes/Desktop/repos/punchbowl/venv/lib/python3.9/site-packages (from regularizepsf) (1.11.2)
Collecting scikit-image
  Using cached scikit_image-0.21.0-cp39-cp39-macosx_12_0_arm64.whl (12.4 MB)
Collecting deepdish
  Using cached deepdish-0.3.7-py2.py3-none-any.whl (37 kB)
Requirement already satisfied: numpy in /Users/jhughes/Desktop/repos/punchbowl/venv/lib/python3.9/site-packages (from regularizepsf) (1.26.0)
Collecting dill
  Using cached dill-0.3.7-py3-none-any.whl (115 kB)
Requirement already satisfied: PyYAML>=3.13 in /Users/jhughes/Desktop/repos/punchbowl/venv/lib/python3.9/site-packages (from astropy->regularizepsf) (6.0.1)
Requirement already satisfied: pyerfa>=2.0 in /Users/jhughes/Desktop/repos/punchbowl/venv/lib/python3.9/site-packages (from astropy->regularizepsf) (2.0.0.3)
Requirement already satisfied: packaging>=19.0 in /Users/jhughes/Desktop/repos/punchbowl/venv/lib/python3.9/site-packages (from astropy->regularizepsf) (23.1)
Collecting tables
  Using cached tables-3.8.0.tar.gz (8.0 MB)
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'error'

  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [11 lines of output]
      <string>:19: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
      ld: library 'hdf5' not found
      clang: error: linker command failed with exit code 1 (use -v to see invocation)
      cpuinfo failed, assuming no CPU features: 'flags'
      * Using Python 3.9.6 (default, Aug 11 2023, 19:44:49)
      * Found cython 3.0.2
      * USE_PKGCONFIG: True
      .. ERROR:: Could not find a local HDF5 installation.
         You may need to explicitly state where your local HDF5 headers and
         library can be found by setting the ``HDF5_DIR`` environment
         variable or by using the ``--hdf5`` command-line option.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

[notice] A new release of pip available: 22.3.1 -> 23.2.1
[notice] To update, run: pip install --upgrade pip
```

**Comment by jmbhughes** [comment]
This seems to be related to deepdish. So, the problem will be solved when we merge from develop.

**Comment by jmbhughes** [comment]
Should be resolved in #63.

**Comment by jmbhughes** [comment]
Fixed!
